conda_recipe_manager/parser/recipe_reader.py

```diff
@@ -704,7 +704,7 @@ def render(self) -> str:
         if self._schema_version == SchemaVersion.V0:
             for key, val in self._vars_tbl.items():
                 # Double quote strings
-                if isinstance(val, str):
+                if isinstance(val, str) and not val.startswith('env.get('):
```

Comment:
Any reason why this line checks against the `(` and the other check doesn't? If we don't make this a regex, maybe we should make the string a constant?
```suggestion
                # See Issue #271 for more details on the `env.get` case.
                if isinstance(val, str) and not val.startswith('env.get('):
```